File: 1_TI/S1/Prog C/TP2/ex5.c
#include<stdio.h>
void main(){
    int a;
    printf("write week: ");
    scanf("%d",&a);
    switch (a)
    {
    case 1 : printf("lundi");
        break;
    case 2 : printf("Mardi");
        break;
    case 3 : printf("Mercredi");
        break;
    case 4 : printf("Jeudi");
        break;
    case 5 : printf("Vendredi");
        break;
    case 6 : printf("Samedii");
        break;
    case 7 : printf("Dimanche");
        break;
    
    default: printf("ERROR");
        break;
    }
}

File: 2_DSI/S1/PHP/TP3/ex5.php
<?php
$emploiDuTemps = array(
    'Lundi' => 'Cours de mathématiques de 9h à 11h',
    'Mardi' => 'Réunion de travail de 14h à 15h',
    'Mercredi' => 'Cours de français de 10h à 12h',
    'Jeudi' => 'Conférence de 16h à 18h',
    'Vendredi' => 'Séance de sport de 17h à 18h',
    'Samedi' => 'Journée libre',
    'Dimanche' => 'Journée libre'
);

$activites = "";

foreach ($emploiDuTemps as $jour => $activite) {
    $activites .= "$jour : $activite\n";
}

echo "<h2>Emploi du temps de la semaine :</h2>";
echo "<pre>$activites</pre>";
?>
